extensions/wikia/SpecialCategoryIntersection/SpecialCategoryIntersection.php

```diff
@@ -216,11 +216,7 @@ private function showResults($out){
 					if(startsWith($key, self::$CAT_PREFIX)){
 						$cat = $wgRequest->getVal($key);
 						if(!empty($cat)){
-							$categories[] = $cat;
-
-							if(!startsWith($cat, $this->CATEGORY_NS_PREFIX)){
-								$html .= "<em>Warning: \"$cat\" does not start with \"{$this->CATEGORY_NS_PREFIX}\".</em><br/>\n";
-							}
+							$categories[] = (!startsWith($cat, $this->CATEGORY_NS_PREFIX) and !startsWith($cat, NS_CATEGORY)) ? $this->CATEGORY_NS_PREFIX.$cat : $cat;
```

Comment:
FANDOM follows [MediaWiki's PHP coding conventions](https://www.mediawiki.org/wiki/CC/PHP). Please ensure your contributions follow this coding style.